registry/remote/example_test.go

```diff
@@ -451,6 +517,55 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// resolve a manifest by tag
+	tag := "formats"
+	descriptor, err := repo.Resolve(ctx, tag)
+	if err != nil {
+		panic(err)
+	}
+	rc, err := repo.Fetch(ctx, descriptor)
+	if err != nil {
+		panic(err)
+	}
+	defer rc.Close() // don't forget to close
+	pulledBlob, err := io.ReadAll(rc)
+	if err != nil {
+		panic(err)
+	}
+	// parse the read content and fetch the descriptors of blobs
+	var formatManifest artifactspec.Manifest
+	if err := json.Unmarshal(pulledBlob, &formatManifest); err != nil {
+		panic(err)
+	}
+	for _, blob := range formatManifest.Blobs {
+		rc, err := repo.Fetch(ctx, ocispec.Descriptor{
+			MediaType: blob.MediaType,
+			Digest:    blob.Digest,
+			Size:      blob.Size,
+		})
+		if err != nil {
+			panic(err)
+		}
+		defer rc.Close()
+		content, err := io.ReadAll(rc)
```

Comment:
We still need to verify the fetched content.

Reply:
Verified by printing out the pulled content.